_includes/footer.html

```diff
@@ -1,21 +1,167 @@
+              <img src="{{ '/images/icons/twitter.svg' | relative_url }}" />
+            </a>
+            <a
+              href="https://www.facebook.com/algorithmia"
+              rel="noopener noreferrer"
+              target="_blank"
+              class="syn-text-secondary syn-link-no-decoration syn-link-animated"
+            >
+              <img src="{{ '/images/icons/facebook.svg' | relative_url }}" />
+            </a>
+            <a
+              href="https://www.linkedin.com/company/algorithmia-inc"
+              rel="noopener noreferrer"
+              target="_blank"
+              class="syn-text-secondary syn-link-no-decoration syn-link-animated"
+            >
+              <img src="{{ '/images/icons/linkedin.svg' | relative_url }}" />
+            </a>
+            <a
+              href="https://www.glassdoor.com/Overview/Working-at-Algorithmia-EI_IE2030295.11,22.htm"
+              rel="noopener noreferrer"
+              target="_blank"
+              class="syn-text-secondary syn-link-no-decoration syn-link-animated"
+            >
+              <img src="{{ '/images/icons/glassdoor.svg' | relative_url }}" />
+            </a>
+          </div>
+        {% endunless %}
+      </div>
+
+      {% unless site.enterprise %}
+        <div class="syn-col syn-col-lg-2 syn-col-md-3">
+          <div class="syn-overline syn-mb-8">Company</div>
+          <div class="syn-mb-8 syn-body-2">
+            <a href="/about" class="syn-text-secondary">About</a>
+          </div>
+          <div class="syn-mb-8 syn-body-2">
+            <a href="/careers" class="syn-text-secondary">Careers</a
+            ><span class="syn-badge theme-secondary syn-ml-8"
+              >We're Hiring!</span
+            >
+          </div>
+          <div class="syn-mb-8 syn-body-2">
+            <a href="https://blog.algorithmia.com" class="syn-text-secondary"
+              >Blog</a
+            >
+          </div>
+          <div class="syn-mb-8 syn-body-2">
+            <a href="/press" class="syn-text-secondary "
+              >Press</a
+            >
+          </div>
+          <div class="syn-mb-8 syn-body-2">
+            <a href="/partners" class="syn-text-secondary"
+              >Partners</a
+            >
+          </div>
+        </div>
 
-    <nav role="navigation" class="bottom-menu pull-right">
-      <ul class="menu-item">
-      {% for link in site.data.footer %}
-        {% if link.url contains 'http' %}
-          {% assign href = site.url %}
-          {% else %}
-          {% assign href = site.url | relative_url %}
-        {% endif %}
-        <li><a href="{{ href }}" {% if link.url contains 'http' %}target="_blank"{% endif %}>{{ link.title }}</a></li>
-      {% endfor %}
-      </ul>
-    </nav><!-- /.bottom-menu -->
+        <div class="syn-col syn-col-lg-2 syn-col-md-3">
+          <div class="syn-overline syn-mb-8">Product</div>
+          <div class="syn-mb-8 syn-body-2">
+            <a
+              to="/pricing"
```

Comment:
This `to` should be an `href`, and `/pricing` is a web client route, not a dev center route. Once this is a standalone container this relative route will 404. We'll need to ensure all links in the footer that route to the web client are absolute.

Reply:
Thanks for catching!